sentences/de/climate_HassClimateGetTemperature.yaml

```
@@ -6,5 +6,7 @@ intents:
           - "Wie hoch ist die Temperatur [<area>]"
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
-          - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie hoch ist die Temperatur (vom Thermostat | der Heizung) <area>"
```

```suggestion
          - "Wie hoch ist die Temperatur ((von dem | vom | des) Thermostat[es] | [von] der Heizung) <area>"
```

would also be great to cover these variations by tests 👍

---

hu, there are tests. i’ll look into it.